src/components/Svg/index.js

```diff
@@ -1,23 +1 @@
-module.exports = require('./Svg');
-
-module.exports.Svg = require('./Svg');
-module.exports.Circle = require('./Circle');
-module.exports.ClipPath = require('./ClipPath');
-module.exports.Defs = require('./Defs');
-module.exports.Ellipse = require('./Ellipse');
-module.exports.G = require('./G');
-module.exports.Image = require('./Image');
-module.exports.Line = require('./Line');
-module.exports.LinearGradient = require('./LinearGradient');
-module.exports.Path = require('./Path');
-module.exports.Pattern = require('./Pattern');
-module.exports.Polygon = require('./Polygon');
-module.exports.Polyline = require('./Polyline');
-module.exports.RadialGradient = require('./RadialGradient');
-module.exports.Rect = require('./Rect');
-module.exports.Stop = require('./Stop');
-module.exports.Symbol = require('./Symbol');
-module.exports.Text = require('./Text');
-module.exports.TextPath = require('./TextPath');
-module.exports.TSpan = require('./TSpan');
-module.exports.Use = require('./Use');
+export { default } from './Svg';
```

Comment:
Is it the es6 syntax or does it only works because we compile it (and it adds a named default export)?

(I’m just curious, haven’t stumbled upon it before)

Reply:
It’s standard syntax; it’s currently the only way to re-export a default as a default.